azuread/resource_user_test.go

```diff
@@ -0,0 +1,145 @@
+package azuread
+
+import (
+	"fmt"
+	"math/rand"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/terraform-providers/terraform-provider-azuread/azuread/helpers/ar"
+)
+
+func TestAccAzureADUser_basic(t *testing.T) {
+	resourceName := "azuread_user.test"
+	id := strconv.Itoa(rand.Intn(9999))
```

Comment:
Could we change this to: 
```suggestion
	id := acctest.RandString(7)
```